lms/static/scripts/frontend_apps/components/SubmitGradeForm.tsx

```diff
@@ -9,12 +9,15 @@ import classnames from 'classnames';
 import { useEffect, useLayoutEffect, useState, useRef } from 'preact/hooks';
 
 import type { StudentInfo } from '../config';
+import { isLTILaunchServerError } from '../errors';
 import type { ErrorLike } from '../errors';
 import { useService, GradingService } from '../services';
 import { useFetch } from '../utils/fetch';
 import { formatGrade, validateGrade } from '../utils/grade-validation';
 import { useUniqueId } from '../utils/hooks';
+import { ErrorState } from './BasicLTILaunchApp';
```

Comment:
Importing from an App doesn't feel right, should a version of that go into errors.ts?